modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -44,12 +44,22 @@ public class UpgradeCatchAllCheck {
 			companyId, 551, "test", start, end);
 	}
 
+	public void method(long companyId, String externalReferenceCode) {
+		_commercePriceListLocalService.fetchByExternalReferenceCode(externalReferenceCode, companyId);
+
```

Comment:
@alberttgomes please add a call from the Utill class as well. To test this case. Should have at least one call from each class and one case to send a message. If before wasn't like that let's follow the new pattern.